cmake/fficonfig.h.in

```diff
@@ -28,14 +28,14 @@
 /* #undef FFI_NO_STRUCTS */
 
 /* Define to 1 if you have `alloca', as a function or macro. */
-#define HAVE_ALLOCA 1
+#cmakedefine HAVE_ALLOCA 1
 
 /* Define to 1 if you have <alloca.h> and it should be used (not on Ultrix).
    */
-#define HAVE_ALLOCA_H 1
+#cmakedefine HAVE_ALLOCA_H 1
 
 /* Define if your assembler supports .cfi_* directives. */
-#define HAVE_AS_CFI_PSEUDO_OP 1
+#cmakedefine HAVE_AS_CFI_PSEUDO_OP 1
```

Comment:
> only being used on m68k

Great detective work.

> current behavior and make a note about i

Sounds great.

> have a way to test and can't sink more time into it atm.

No need to test, adding a comment to summarize your investigations would be great :+1: